File: ass59/Program5.java
/*
5. Write Java program which accept String from user and display below pattern. 
Input :  Hello 
Output : H   
         H e   
         H e l   
         H e l l   
         H e l l o   
         H e l l   
         H e l    
         H e l   
         H e   
         H
*/
import java.lang.*;
import java.util.*;
class DemoString
{
	int WordFrequency(String str)
	{
		char arr[] = str.toCharArray();
		int size = arr.length;
        int i = 0, j= 0;
        
        for(i=0; i<size-1; i++)
        {
            for(j=0; j<=i; j++)
            {
                System.out.print(""+arr[j]);
            }
            System.out.print("\n");
        }
        for(i=size-1; i>=0; i--)
        {
            for(j=0; j<=i; j++)
            {
                System.out.print(""+arr[j]);
            }
            System.out.print("\n");
        }
        return 0;
    }
    
}
class Program5
{
	public static void main(String arg[])
	{
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter string");

        String str = sobj.nextLine();
        sobj.close();
		DemoString dobj = new DemoString();

		dobj.WordFrequency(str);
	}
}

File: ass13/program5.c
/*
Accept number from user and display below pattern. 
Input :  8 
Output : 2 4 6 8 10 12 14 16 
*/
#include<stdio.h>

void Pattern(int iNo)
{
    int iCnt = 0,iRet=0;
    
    if(iNo < 0)         // Updator
    {
        return;
    }
    for(iCnt = 1;iCnt <= iNo;iCnt++)
    {
         printf("%d\n",iCnt*2);
    }
}   

int main()
{
    int iValue = 0;
    
    printf("Enter number\n");
    scanf("%d",&iValue);
    
    Pattern(iValue);
    
    return 0;
}